.docforge/manifest.yaml

```
@@ -1,13 +1,6 @@
-{{- $vers := Split .versions "," -}}
-{{ $defaultBranch := (index $vers 0) }}
 structure:
 - name: _index.md
-  source: https://github.com/gardener/gardener-extension-provider-openstack/blob/{{$defaultBranch}}/README.md
+  source: /README.md
 - name: docs
-  nodes:
-  - nodesSelector:
-      path: https://github.com/gardener/gardener-extension-provider-openstack/tree/{{$defaultBranch}}/docs
-links:
-  downloads:
-    scope:
-      "gardener/gardener-extension-provider-openstack/(blob|raw)/(.*)/docs": ~
+  nodesSelector:
+      path: /docs
```

This does not look like a properly formatted yaml:
```suggestion
  nodesSelector:
    path: /docs
```

---

yes, I added a fixing commit. Thanks for noting